python/semantic_kernel/planning/basic_planner.py

```diff
@@ -194,6 +194,9 @@ async def execute_plan_async(self, plan: Plan, kernel: Kernel) -> str:
         subtasks = generated_plan["subtasks"]
 
         for subtask in subtasks:
+            if debug_print:
```

Comment:
I've found these useful, to watch what's happening through a plan's execution, but I'm happy to remove these changes from the PR.